node-js-basics-101/node_modules/parseuri/test.js

```diff
@@ -0,0 +1,51 @@
+var assert = require('better-assert');
+var expect = require('expect.js');
+var parseuri = require('./index.js');
+
+describe('my suite', function(){
+	it('should parse an uri', function () {
+    var http = parseuri('http://google.com')
+      , https = parseuri('https://www.google.com:80')
+      , query = parseuri('google.com:8080/foo/bar?foo=bar')
+      , localhost = parseuri('localhost:8080')
```

Comment:
__[jshint]__
* Misleading line break before ','; readers may interpret this as an expression boundary.

<!-- Comment by [SideCI](https://sideci.com) -->